src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(SEPARATOR = ',') {
     // write code here
+    let RESULT = '';
+
+    for (let i = 0; i < this.length; i++) {
+      const CURRENT_NUMBER = this[i];
+
+      if (CURRENT_NUMBER !== null && CURRENT_NUMBER !== undefined) {
+        RESULT += CURRENT_NUMBER;
+      }
```

Comment:
Are you sure this is correct?

Please read method join spec and check out how it handles sparse arrays, null/undefined values

(ex. see here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join)